src/main/java/org/opensrp/web/rest/PlanResource.java

```diff
@@ -186,4 +190,22 @@ else if (sName == null && field.getName().equals(fieldName)) {
 		}
 		return false;
 	}
+
+
+	static class PlanSyncRequestWrapper {
+		@JsonProperty
+		private String operational_area_id;
```

Comment:
Change this to `List<String>` as was before so that one sends a list on ids instead of using strings which we then split using a comma, e.g the request will be looking like 
```
{
  "operational_area_id": [
    "450fc15b-5bd2-468a-927a-49cb10d3bcac",
    "304cbcd4-0850-404a-a8b1-486b02f7b84d"
  ],
  "serverVersion": 0
}
```